### PR TITLE
Move stuff from motion to match original picture

### PR DESCRIPTION
**Pull request by paddyroddy**

### CODE REVIEW

File: btrack/napari/widgets/_hypothesis.py [hunk 1]
```diff
@@ -22,7 +20,7 @@ def _create_hypotheses_widgets() -> dict[str, tuple[str, QtWidgets.QWidget]]:
     ]
 
     widget = QtWidgets.QListWidget()
-    widget.addItems(hypotheses)
+    widget.addItems([f"{h.replace('_', '(')})" for h in hypotheses])
```
Comment (paddyroddy) [inline btrack/napari/widgets/_hypothesis.py:23]: I would have liked to make it bold here `<b>P</b>()` kind of thing but it seems non-straightforward for `QListWidgetItem`

File: btrack/napari/widgets/_motion.py [hunk 2]
```diff
@@ -3,12 +3,6 @@
 from qtpy import QtWidgets
 
 
-def _make_label_bold(label: str) -> str:
-    """Generate html for a bold label"""
-
-    return f"<b>{label}</b>"
```
[the remark this answers is not in the document]
Comment (p-j-smith) [reply btrack/napari/widgets/_motion.py:9]: Yeah that makes sense

**Review by p-j-smith — APPROVED** [review]
looking good

File: btrack/napari/widgets/_hypothesis.py (hunk 1 above)
Comment (paddyroddy) [reply btrack/napari/widgets/_hypothesis.py:23]: So I think it is possible, but you have to do something like this https://stackoverflow.com/a/53572278/7359333 which seems very involved

**Pull request merged**